### DISCUSSION TITLE
Running two "normal" and one tumor sample for a single patient

### DISCUSSION THREAD

**Discussion by smahesh12** [uclahs-cds/metapipeline-DNA#109] [General]
I noticed on README.md that metapipeline-DNA can handle multiple tumor samples per patient, but can handle exactly one normal sample per patient. How should I go about analyzing two "normal" samples for a patient (e.g. "Blood" and "Adjacent Normal") with one tumor sample ("Primary")? Would you suggest submitting two jobs with two separate input CSVs - one with Blood (N) and Primary (T), and another with Adjacent Normal (N) and Primary (T)?

@uclahs-cds/meta-pipeline-wg

**Comment by yashpatel6** [comment]
In terms of logistics with the metapipeline, it would require either submitting separate jobs or having one job with both pairs but with each pair having a different patient ID (ex. `patient1-blood` for blood and tumor samples and `patient1-adjecent` for adjacent and tumor samples - the tumor sample would be listed twice in this case, once for each "patient").

**Reply by smahesh12** [reply]
Sounds good, that makes sense; thank you for the explanation!

**Comment by tnyamaguchi** [comment]
It depends on the goal of your project and the sample condition but if you treat the adjacent normal sample as normal, you would have to submit two jobs as you described. 

Let's say if there is any concern about tumor contamination in the adjacent normal sample, you could treat the adjacent normal sample as a tumor sample and see if you identify any somatic driver mutations in your adjacent normal compared to the blood sample. 

Let's say if you find high cross-individual contamination in the blood sample, you would have to use the adjacent normal as normal and drop the blood sample.

You could potentially ignore the adjacent normal sample if you are just interested in somatic mutations in the primary tumor sample and the blood sample is clean.

**Reply by smahesh12** [reply]
I see - it makes more sense to me now, and I have a better idea of how each sample should be used in our project. Thanks for the detailed response!
